config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,42 @@
-create table bad_words (
-    word character varying(40) not null
+create table ta_users (
+    username character varying(40) not null primary key,
+    password character varying(60) not null,
+    email character varying(254) not null,
+    joined timestamp with time zone not null default current_timestamp,
+    lastlogin timestamp with time zone not null default current_timestamp,
+    admin boolean not null default false
 );
+alter table ta_users owner to postgres;
 
-
-alter table bad_words owner to postgres;
-
-
-create table banned_ips (
-    ip character varying(40) not null,
-    ban_till timestamp without time zone
+create table bad_words (
+    word character varying(40) not null primary key
 );
-
-
-alter table banned_ips owner to postgres;
+alter table bad_words owner to postgres;
 
 
 create table banned_macs (
-    mac character varying(40) not null,
-    ban_till timestamp without time zone
+    mac character(34) not null primary key,
+    ban_till timestamp with time zone
 );
-
-
 alter table banned_macs owner to postgres;
 
 
 create table banned_usernames (
-    username character varying(40) not null,
-    ban_till timestamp without time zone
+    username character varying(40) not null primary key, -- references ta_users on delete cascade on update cascade,
```

Comment:
@DanVanAtta Nevermind. The code does allow this described behaviour, moving into a comment